From patchwork Thu Oct 10 02:52:49 2019
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 8bit
X-Patchwork-Submitter: "richard.peng@oppo.com" <richard.peng@oppo.com>
X-Patchwork-Id: 11182547
Return-Path: <SRS0=OAn+=YD=vger.kernel.org=kvm-owner@kernel.org>
Received: from mail.kernel.org (pdx-korg-mail-1.web.codeaurora.org
 [172.30.200.123])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id AC889112B
	for <patchwork-kvm@patchwork.kernel.org>;
 Thu, 10 Oct 2019 03:01:31 +0000 (UTC)
Received: from vger.kernel.org (vger.kernel.org [209.132.180.67])
	by mail.kernel.org (Postfix) with ESMTP id 971E2206C0
	for <patchwork-kvm@patchwork.kernel.org>;
 Thu, 10 Oct 2019 03:01:31 +0000 (UTC)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S1732448AbfJJDBa (ORCPT
        <rfc822;patchwork-kvm@patchwork.kernel.org>);
        Wed, 9 Oct 2019 23:01:30 -0400
Received: from m17617.mail.qiye.163.com ([59.111.176.17]:10034 "EHLO
        m17617.mail.qiye.163.com" rhost-flags-OK-OK-OK-OK) by vger.kernel.org
        with ESMTP id S1726621AbfJJDBa (ORCPT <rfc822;kvm@vger.kernel.org>);
        Wed, 9 Oct 2019 23:01:30 -0400
X-Greylist: delayed 522 seconds by postgrey-1.27 at vger.kernel.org;
 Wed, 09 Oct 2019 23:01:29 EDT
Received: from P80254710 (unknown [121.12.147.249])
        by m17617.mail.qiye.163.com (Hmail) with ESMTPA id 3A9C7261AAB;
        Thu, 10 Oct 2019 10:52:45 +0800 (CST)
Date: Thu, 10 Oct 2019 10:52:49 +0800
From: "richard.peng@oppo.com" <richard.peng@oppo.com>
To: pbonzini <pbonzini@redhat.com>, rkrcmar <rkrcmar@redhat.com>
Cc: kvm <kvm@vger.kernel.org>,
        linux-kernel <linux-kernel@vger.kernel.org>
Subject: [PATCH] kvm/x86 : Replace BUG_ON(1) with BUG()
X-Priority: 3
X-GUID: DC6DBE28-1918-4988-81E0-F1DEDA6BAD10
X-Has-Attach: no
X-Mailer: Foxmail 7.2.10.151[cn]
Mime-Version: 1.0
Message-ID: <201910101052488242114@oppo.com>
X-HM-Spam-Status: e1kfGhgUHx5ZQUtXWQgYFAkeWUFZVk9VTE5NS0tKT0xJT0xISk9ZV1koWU
        FJSUtLSjdXWS1ZQUlXWQkOFx4IWUFZNTQpNjo3JCkuNz5ZBg++
X-HM-Sender-Digest: e1kMHhlZQR0aFwgeV1kSHx4VD1lBWUc6MSI6Tgw4QzlNFUw9AywJGh0a
        NhxPFEtVSlVKTkxLTUxOQk1OTE5JVTMWGhIXVQkSGBMaCR9VCx4VHDsUCwsUVRgUFkVZV1kSC1lB
        WUpJSlVKSVVKT0xVSU9CWVdZCAFZQUpNSk03Bg++
X-HM-Tid: 0a6db3953e9b9375kuws3a9c7261aab
Sender: kvm-owner@vger.kernel.org
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org

Signed-off-by: Peng Hao <richard.peng@oppo.com>
---
 arch/x86/kvm/vmx/nested.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

-- 
2.7.4

diff --git a/arch/x86/kvm/vmx/nested.c b/arch/x86/kvm/vmx/nested.c
index e76eb4f..d0e6c40 100644
--- a/arch/x86/kvm/vmx/nested.c
+++ b/arch/x86/kvm/vmx/nested.c
@@ -4945,8 +4945,7 @@ static int handle_invept(struct kvm_vcpu *vcpu)
 	 */
 		break;
 	default:
-		BUG_ON(1);
-		break;
+		BUG();
 	}
 
 	return nested_vmx_succeed(vcpu);
